src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/Activities/CalculateEarnings.cs

```diff
@@ -21,9 +21,9 @@ public CalculateEarningsActivity(ICommandDispatcher commandDispatcher, ILogger<C
         [FunctionName(nameof(CalculateEarningsActivity))]
         public async Task Update([ActivityTrigger] CalculateEarningsInput input)
         {
-            _logger.LogInformation("Calculating Earnings for apprenticeship incentive id {apprenticeshipIncentiveId}, uln {uln}", input.ApprenticeshipIncentiveId, input.Uln);
+            _logger.LogDebug("Calculating Earnings for apprenticeship incentive id {apprenticeshipIncentiveId}, uln {uln}", input.ApprenticeshipIncentiveId, input.Uln);
             await _commandDispatcher.Send(new CalculateEarningsCommand(input.ApprenticeshipIncentiveId));
```

Comment:
Shouldn't the log be in a decorator of the logger be a decorator of the _commandDispatcher?